serverlessworkflow/modules/ROOT/pages/cloud/deploying-on-kubernetes.adoc

```diff
@@ -0,0 +1,96 @@
+= Deploying your Serverless Workflow application on Kubernetes
+:description: Deploying Serverless Application on Kubernetes
+:keywords: kogito, workflow, quarkus, serverless, kn, kubectl
+:table-caption: Data Set
+// envs for common content
+:environment_prereq: Kubernetes Cluster is available
+:kubectl_prereq: `kubectl` command-line tool is installed.
+:cluster_kind: Kubernetes with Knative is ready
+:knative_procedure: <<proc-verify-knative-kubernetes, Verifying Knative on Kubernetes>>
+:deploy_application: on Kubernetes
+:k8s_registry: quay.io/mynamespace
+:registry: target
+:container_push: Property to whether or not push images
+
+// links
+:knative_kubernetes_url: https://knative.dev/docs/install/
+:verify_knative_url: https://knative.dev/docs/serving/knative-kubernetes-services/#before-you-begin
+:quay_io_url: https://quay.io/
+:k8s_pull_secret_url: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+:knative_domain_dns_url: https://knative.dev/docs/install/operator/knative-with-operators/#configure-dns
+
+
+This document describes how to deploy a workflow application using a local Kubernetes cluster, along with a procedure
+to run the Knative platform.
+
+// shared pre req
+include::common/_prerequisites.adoc[]
+
+Before proceeding, make sure that you have access to the Kubernetes cluster with Knative available.
+
+[[proc-verify-knative-kubernetes]]
+== Verifying Knative on Kubernetes
+
+To make sure Knative is available, it can be checked with the commands below:
+
+.Verify if Knative is available
+[source,shell]
+----
+kubectl get services -n knative-serving
```

Comment:
I am avoiding to add the shell indicator because when user copy it using the copy button in the rendered docs user will need to paste and remove the $ character.

@kaldesai do we have a pattern about it?